### PR TITLE
Enforce use of secure LDAP

### PR DESCRIPTION
**Pull request by amrutagokhale**
Enforce use of secure LDAP and reject connection attempt if secure LDAP has not been enabled.

@anusudarsan Take a look, please.

### CODE REVIEW

**Comment by amrutagokhale** [comment]
Tested manually.

File: presto-main/src/main/java/com/facebook/presto/server/security/ServerSecurityModule.java [hunk 1]
```diff
@@ -66,6 +66,10 @@ private void setupLdap(LdapServerConfig ldapServerConfig, Binder binder)
             checkState(ldapServerConfig.getUserObjectClass() != null, "Missing property 'authentication.ldap.user-object-class'");
         }
 
+        if (ldapServerConfig.getLdapUrl().startsWith("ldap://")) {
```
Comment (anusudarsan) [inline presto-main/src/main/java/com/facebook/presto/server/security/ServerSecurityModule.java:69]: can you check if sth like `ldap.url=ldapserver:389` works? In this case it doesnt start with ldaps:// but is still using not 
secure LDAP

Comment (amrutagokhale) [reply presto-main/src/main/java/com/facebook/presto/server/security/ServerSecurityModule.java:69]: With `ldap.url=ldapserver:389`, it gives `MalformedURLException`.
With `ldap.url=ldaps://ldapserver:389`, it gives `SSLHandshakeException`.

Comment (anusudarsan) [reply presto-main/src/main/java/com/facebook/presto/server/security/ServerSecurityModule.java:69]: thanks for checking. LGTM

**Pull request merged**